docker-compose-prod.yml

```diff
@@ -13,30 +13,38 @@ services:
       retries: 5
```

Comment:
we need 2 files:
- `docker-compose.yml` for local development
- `docker-compose-prod.yml` for production deployment.